test/integration/lib/render.js

```diff
@@ -175,6 +175,10 @@ async function runTest(t) {
 
         // Disable anisotropic filtering on render tests
         map.painter.context.extTextureFilterAnisotropicForceOff = true;
+        // Disable globe antialiasing on render tests expect for antialiasing test
+        if (!currentTestName.startsWith('render-tests/globe/globe-antialiasing')) {
```

Comment:
It's also possible to also use instead a render test metadata option, and check against it similarly as above `if (options.collisionDebug)` for example. But this feels unique enough that it's ok, and we could go ahead with switching to an option if needed.